h2o-py/h2o/frame.py

```diff
@@ -1221,35 +1221,44 @@ def structure(self):
                 print("num {}".format(" ".join(it[0] if it else "nan" for it in h2o.as_list(self[:10, i], False)[1:])))
 
 
-    def as_data_frame(self, use_pandas=True, header=True):
+    def as_data_frame(self, use_pandas=True, header=True, na_value=''):
```

Comment:
if we set it automatically, then it won't be backward consistent. Till this change, the NA values are missing from the Pandas (see https://0xdata.atlassian.net/browse/PUBDEV-4723?filter=-1). However it can be considered as a bug, which will be fixed by this PR. Should I set it automatically and remove the na_value parameter?